### ISSUE TITLE
[FEATURE REQUEST] - Random

### ISSUE THREAD

**Issue by KendallDoesCoding** [KendallDoesCoding/Choose-Your-Own-Adventure-Game#19]
Right now it is made so that for example, the user goes left they go to chapter river, and if they go right they go to chapter bridge. But I want it to be random for this and throughout the game. Because, if a user keeps on playing it they will understand how to win and it's no longer going to be fun.

It's easy to implement this using choice() (import random) is what I've heard.

**Comment by SahilSinghTomar** [comment]
in this feature we only need to randomize the left and right path. am i right

**Comment by KendallDoesCoding** [comment]
> in this feature we only need to randomize the left and right path. am i right
> 
> like if user chooses left he can go both river or bridge chapter

As a start, yes.
But also we can randomize it for everything, for example for walk it goes ahead and swim you die so you can make it so that sometimes even for walk you die but then for the game over message we will need an option to type a message depending on how they died because otherwise it makes no sense to drown while walking lol.

**Comment by KendallDoesCoding** [comment]
@SahilSinghTomar - would you like to work on this issue?

**Comment by SahilSinghTomar** [comment]
right now i can fix only the first part 
is it ok??

**Comment by KendallDoesCoding** [comment]
> right now i can fix only the first part is it ok??



> right now i can fix only the first part is it ok??

Yup, ofcourse. The second part would take some work from me as well as I have to change the game-over messages and stuff.

**Comment by Zhreyu** [comment]
Hey can I work on this, if its not fixed yet?

**Comment by KendallDoesCoding** [comment]
> Hey can I work on this, if its not fixed yet?

We have to double check if it's fixed or not. I'm not sure.

**Comment by swetank-367** [comment]
Hey Can I work on this ? if it is not solved yet.
